js/ClockCtrl.js

```diff
@@ -28,10 +28,7 @@ var timeFilter = function() {
         };
 }
 
-angular.module('Clock',['ngStorage'])
-    .filter('time', timeFilter)
-    .factory('$audio',[
-        function() {
+var AudioService = function() {

```

Comment:
Better to use `function AudioService` (leave out the var), because it will show nicer names in e.g. stack traces (it's now an anonymous function).